app-shells/fzf/fzf-0.16.2.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+GOLANG_PKG_IMPORTPATH="github.com/junegunn"
+GOLANG_PKG_BUILDPATH="/src/${PN}"
+GOLANG_PKG_TAGS="release"
+GOLANG_PKG_HAVE_TEST=1
+GOLANG_PKG_USE_CGO=1
+
+GOLANG_PKG_DEPENDENCIES=(
+	"github.com/junegunn/go-isatty:66b8e73"
+	"github.com/junegunn/go-runewidth:63c378b"
+	"github.com/junegunn/go-shellwords:35d512a"
+	"github.com/golang/crypto:f6b343c -> golang.org/x"
```

Comment:
Please, can you update `github.com/junegunn/go-shellwords` to `33bd8f1`, and `github.com/golang/crypto` to `854ae91`?